app/database/queries/breeze_buddy/data_source.py

```
@@ -0,0 +1,145 @@
+Layer 1 of the three-layer DB pattern: pure SQL, no DB calls.
+Returns (sql_string, values_list) tuples consumed by run_parameterized_query().
+"""
+
+from datetime import datetime
+from typing import Any, List, Optional, Tuple
+
+DATA_SOURCE_TABLE = "data_source"
+
+
+def insert_data_source_query(
+    id: str,
```

Ruff (0.15.15)

[error] Function argument `id` is shadowing a Python builtin

(A002)